demo-automation-java/src/test/java/com/example/tests/BaseTest.java

```diff
@@ -31,14 +43,6 @@ public void setUp() throws Exception {
         } catch (Exception e) {

```

Comment:
Are you apply implicit wait in here?
But I haven't seen it applied here, it could be "driver.manage().timeouts().implicitlyWait(implicitWait, TimeUnit.SECONDS);" instead "new WebDriverWait()"

Reply:
There is a conflict in BaseTest.java in Develop so i just chose one and delete the other